modules/acm_sku/src/ProductOptionsManager.php

```diff
@@ -188,12 +190,16 @@ protected function createProductOption($langcode, $option_id, $option_value, $at
         $this->logger->critical('Product option "@option" wasn\'t saved. Try again later please.', ['@option' => $option_value]);
       }
     }
+
+    return $term;
   }
 
   /**
-   * {@inheritdoc}
+   * Synchronize all product options.
```

Comment:
This one is in the interface, so `@inheritdoc` should be good enough

Reply:
reverted